README.md

```diff
@@ -1 +1,139 @@
+
+    Make simple calculations.
+    Read a random math-related quote.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>FrontEnd</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Test</summary>
+  <ul>
+    <li><a href="https://jestjs.io/docs/tutorial-react">jest</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features 
+
+A react logo
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo 
+
+- [Live Demo Link](coming soon)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started 
+
+To get a local copy up and running follow these simple example steps.
+## Prerequisites
+
+  1. You need to have Code Editor Installed
+  2. Make sure Nodejs and NPM is installed in your system for linters and libraries
+
+## Setup
+
+1. Clone the repo in your local machine:
+
+```bash
+git clone git clone https://github.com/khanetmoi/math-magician-app
+```
+2. Open terminal and cd into project and install the dependencies
+```bash
+cd Math-Magician
+npm install
+```
+
+3. run the command to open your index.html automatically in your brower:
+```bash
+npm run build
+npm start
+```
+
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Inayat-khan**
+
+- GitHub: [@githubhandle](https://github.com/Khanetmoi)
+- Twitter: [@twitterhandle](https://twitter.com/InayatVan)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/khan-bitsindou-b37178228/)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features 
+
+  * Calculator UI - A simple calculator interface
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing 
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Khanetmoi/math-magician-app/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support 
+
+  * Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments 
+
+  * Hat tip to anyone whose code was used
+  * Inspiration
+  * etc
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```

Comment:
- [OPTIONAL] Kindly remove  React.js in the Client Tech Stack and Express.js in the Server Tech Stack since we're not using them in the project 👇🏻 


![image](https://user-images.githubusercontent.com/79058364/220674010-44428ea3-610f-4e71-b438-6cacaa4dba8f.png)